pallets/marketplace/src/benchmarking.rs

```diff
@@ -58,7 +54,7 @@ where
 
 pub fn job_registration_with_reward<T: Config>(
     script: Script,
-    cpu_milliseconds: u128,
+    duration: u64,
```

Comment:
I assume that this is some design change.

Reply:
Yes and now, the duration moved into Schedule and the naming is consistently without the unit